lua/tests/infoview/autoopen_spec.lua

```diff
@@ -33,7 +33,7 @@ describe('infoview', function()
     assert.buf.created.tracked()
     assert.win.created.tracked()
     vim.api.nvim_command("edit " .. fixtures.lean3_project.some_existing_file)
-    assert.buf.left.created({infoview.get_current_info().bufnr}).tracked()
+    assert.buf.left.created({infoview.get_current_infoview().info.bufnr}).tracked()
     assert.no_buf_track.initopened.infoview()
```

Comment:
Yeah, this `no_track_buf` stuff is a bit less than ideal, I shouldn't have had to come back to modify this. I think a solution would be something like `assert.buf.left.tracked_pending()` to store an assertion state that is merged in on the next `assert. ... .tracked()` run. Will do this soon!